### ISSUE TITLE
[BUG] Groups - "Create Post" inaccessible to assistive technologies

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5770]
**Describe the bug**
Screen reader users are unable to input text into the "Post" field when creating a new Group post. 

This violates[ WCAG 2.1.1 Keyboard ](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable)

**To Reproduce**
Using TalkBack or VoiceOver
1. Go to a Group that user is a member of
2. Click on '+’
3. Select ‘Create Post’
4. Complete Schedule and Subject fields
5. Gesture to 'Post' to input text

**Expected behavior**
 Screen reader user is able to input text in the "Post" field

**Actual behavior**
  Screen reader user is NOT able to input text in the "Post" field

**Screenshots**

https://github.com/user-attachments/assets/3f6d9521-b9db-4ef1-82c7-6cc11237aaa8

**Smartphone (please complete the following information):**
- Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.51

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.51

**Additional context**

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/f4113a92-4d1a-41da-b776-271ab21e47d6)

**Comment by TodorBachvarov** [comment]
@acovher2, this should be fixed in version 7.3.52 / 8.0.8

**Comment by acovher2** [comment]
Verified as fixed in 7.3.52
